Update intro.md (#504)
testing-library/testing-library-docs: docs/react-testing-library/intro.md

> [`DOM Testing Library`](dom-testing-library/intro.md) which is where most of
> the logic behind the queries is.

## Tutorials

Have a look at the "What is React Testing library?" video below for an
introduction to the library.

<a href="https://youtu.be/JKOwJUM4_RM">
  <img width="200px" alt="what is react testing library" src='https://img.youtube.com/vi/JKOwJUM4_RM/0.jpg' style="margin-left: 0" />
</a>

> If you prefer reading over watching, check this [tutorial for React Testing Library](https://www.robinwieruch.de/react-testing-library).

Reviewer: Every video on https://TestingJavaScript.com has enhanced transcripts with inline images and code, so people can consume it via reading as well. So have about changing this line to:

```suggestion
Also, don't miss this [tutorial for React Testing Library](https://www.robinwieruch.de/react-testing-library).
```

Author: Ah sure! I just saw the video by @stolinski there and I wasn't sure whether he provides transcripts as well :) But I bet he has them too! 🚀

Reviewer: Oh 😅 I didn't look carefully enough and thought this was referring to my course 😅
